lib/mkDummySrc.nix

```diff
@@ -125,6 +125,7 @@ let
   dummyrs = args.dummyrs or (writeText "dummy.rs" ''
     #![allow(dead_code)]
     #![cfg_attr(any(target_os = "none", target_os = "uefi"), no_std)]
+    #![cfg_attr(any(target_os = "none", target_os = "uefi"), no_main)]
```

Comment:
Why do we need `no_main` if we have a(n empty) main function further down?

Reply:
I am honestly not sure, the `noStd` test fails without this statement with error regarding missing "`start` lang_item"
Annotating `main` with `start` would require a nightly feature

Reply:
Hmm, that's weird, but if it works it works (we can always clean this up later, so no blocker here)